web/marketing/pages/referral-offers.html

```diff
@@ -7,14 +7,30 @@
         <div class="col col-heading">Name</div>
         <div class="col col-heading">Give Credit</div> 
         <div class="col col-heading">Award Credit</div>
-        <div class="col col-heading">Referrals&nbsp;Used</div>
-        <div class="col col-heading">Redeemable&nbsp;Capacity</div>
+        <div class="col col-heading">Referrals Used</div>
+        <div class="col col-heading">Redeemable Capacity</div>
         <div class="col col-heading">Created</div>
         <div class="col col-heading">Expiration</div>
         <div class="col col-heading">Status</div>
     </div><hr>
     <div class="row offer-heading ">
-        <p class="offer-type">default&nbsp;offer</p>
+        <p class="offer-type">default offer</p>
+    </div>
+    {{range .ReferralOffers}}
+        {{if eq .Status 1}}
+            <div class="row data-row">
+                <div class="col ml-3">{{.Name}}</div>
+                <div class="col">${{.InviteeCreditInCents}}</div> 
+                <div class="col">${{.AwardCreditInCents}}</div>
+                <div class="col">{{.NumRedeemed}}</div>
+                <div class="col">&#8734;</div>
+                <div class="col">{{printf "%.10s" .CreatedAt}}</div>
+                <div class="col">&#8734;</div>
+            </div><hr>
+        {{end}}
+    {{end}}
+    <div class="row offer-heading ">
+        <p class="offer-type">current offer</p>
     </div>
     {{range .ReferralOffers}}
         {{if eq .Status 2}}
```

Comment:
`{{if .Status.IsXYZ}}`, you can call methods on types to avoid these magic number.s